packages/valory/skills/abstract_round_abci/models.py

```diff
@@ -443,10 +441,12 @@ def setup(self) -> None:
                     cross_period_persisted_keys=self.abci_app_cls.cross_period_persisted_keys,
                 )
             ),
-            consensus_params,
             self.context.logger,
         )
-        self.initial_tm_configs = dict.fromkeys(self.synchronized_data.all_participants)
+        if not self.context.is_abstract_component:
```

Comment:
Why this change?

Reply:
Abstract components do not need to specify `all_participants` or set the `initial_tm_configs`.

Therefore, in this PR when I had to add the new `consensus_params` in the setup of the skills' configurations, I decided to remove the setup from the abstract skills instead of adding the new one, since they were all unnecessary.